client/me/purchases/purchases-site/style.scss

```diff
@@ -45,3 +45,11 @@
 		padding: 11px 24px;
 	}
 }
+
+@include breakpoint( ">960px" ) {
+	.purchases-list {
+		.action-card__illustration {
```

Comment:
Probably doesn't need to be nested. Should just be `.purchase-list .action-card_illustration`

Reply:
done, thanks :)